tests/constants/data.csv

```diff
@@ -0,0 +1,10 @@
+role,name,team
+admin,admin,
+tutor,tutor1,CS2103T-W1-1
```

Comment:
This csv seems duplicated in `data.constants.ts` as well? Shall we remove either?

Reply:
Removed